DataConnectors/Trend Micro/TrendMicroXDR_REST_API_Connector.json

```diff
@@ -0,0 +1,134 @@
+        },
+        {
+            "title": "",
+            "description": "**STEP 1 - Configuration steps for the Trend Micro XDR API**\n\n [Follow these instructions](https://docs.trendmicro.com/en-us/enterprise/trend-micro-xdr-help/ObtainingAPIKeys) to create an account and an API authentication token." 
+        },
+        {
+            "title": "",
+            "description": "**STEP 2 - Use the below deployment option to deploy the connector and the associated Azure Function**\n\n>**IMPORTANT:** Before deploying the Trend Micro XDR connector, have the Workspace ID and Workspace Primary Key (can be copied from the following), as well as the Trend Micro XDR API Authorization Token, readily available.", 
+            "instructions":[
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Workspace ID"
+                    },
+                    "type": "CopyableLabel"
+                },
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "PrimaryKey"
+                        ],
+                        "label": "Primary Key"
+                    },
+                    "type": "CopyableLabel"
+                }
+            ]
+        },
+        {
+            "title": "Azure Resource Manager (ARM) Template Deployment",
+            "description": "This method provides an automated deployment of the Trend Micro XDR connector using an ARM Tempate.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Ftrendmicroxdr.blob.core.windows.net%2Farm-template%2Fazuredeploy_TrendMicroVisionOne_API_FunctionApp.json)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n3. Enter a unique **Function Name**, **Workspace ID**, **Workspace Key**, **API Token** and **Region Code**. \n - Note: Provide the appropriate region code based on where your Trend Micro XDR instance is deployed: us, eu, au, in, sg, jp  \n - Note: If using Azure Key Vault secrets for any of the values above, use the`@Microsoft.KeyVault(SecretUri={Security Identifier})`schema in place of the string values. Refer to [Key Vault references documentation](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) for further details. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```

Comment:
Instead of custom storage, please point to the arm template in the GitHub master branch. 

Please replace "https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Ftrendmicroxdr.blob.core.windows.net%2Farm-template%2Fazuredeploy_TrendMicroVisionOne_API_FunctionApp.json"

with shortlink:  https://aka.ms/sentinel-trendmicroxdr-azuredeploy, we will be responsible to create arm deployment link pointing to the azuredeploy_TrendMicroVisionOne_API_FunctionApp.json file